rotkehlchen/tests/unit/decoders/test_curve.py

```diff
@@ -1725,3 +1735,307 @@ def test_no_zap_event(ethereum_transaction_decoder, ethereum_accounts):
         ),
     ]
     assert events == expected_events
+
+
+@pytest.mark.vcr()
+@pytest.mark.parametrize('polygon_pos_accounts', [['0x1c362DFE864a4c4b3311eC97bf0b8320CB0a4952']])
+def test_curve_deposit_polygon(database, polygon_pos_inquirer, polygon_pos_accounts):
+    tx_hash = deserialize_evm_tx_hash('0x6cb9d7ceb55a1063c17b58cb643e699525ca6037e711c34283cf0f3d6e81716e')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=polygon_pos_inquirer,
+        database=database,
+        tx_hash=tx_hash,
+    )
+    timestamp, pool_address = TimestampMS(1718015936000), string_to_evm_address('0x445FE580eF8d70FF569aB36e80c647af338db351')  # noqa: E501
+    deposit_amount, approve_amount, received_amount, gas_fees = '2.12', '0.88', '1.87538744644899407', '0.0127938000149261'  # noqa: E501
```

Comment:
Combine all in same line. Already need noqa for each line so just combine them. Same in all other tests